src/Adapter/PlatformServices.Shared/netstandard1.0/Services/ns10FileOperations.cs

```diff
@@ -16,12 +16,6 @@ namespace Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices
     /// </summary>
     public class FileOperations : IFileOperations
     {
-        private static MethodInfo methodGetNavigationData;
-        private static PropertyInfo propertyFileName;
-        private static PropertyInfo propertyMinLineNumber;
-        private static Type typeDiaSession;
-        private static Type typeDiaNavigationData;
-
         /// <summary>
         /// Initializes static members of the <see cref="FileOperations"/> class.
         /// </summary>
```

Comment:
Should the initialization now move into the new operations class? Doesn't make much sense for this to be aware of diasession type anymore.